README.md

```diff
@@ -46,4 +46,68 @@ const STARGATE_URL = "https://stargate.cosmos.network"
 const api = API(STARGATE_URL)
 
 const validators = await api.validators()
+```
+
+### Create a sign message to sign with on a Ledger or with any other signer
+
+```
+const { signWithPrivateKey } = require('@lunie/cosmos-keys');
+const { createSignMessage } = require('@lunie/cosmos-api');
+
+const stdTx = {
+  msg: [
+    {
+      type: `cosmos-sdk/Send`,
+      value: {
+        inputs: [
+          {
+            address: `cosmos1qperwt9wrnkg5k9e5gzfgjppzpqhyav5j24d66`,
+            coins: [{ denom: `STAKE`, amount: `1` }]
+          }
+        ],
+        outputs: [
+          {
+            address: `cosmos1yeckxz7tapz34kjwnjxvmxzurerquhtrmxmuxt`,
+            coins: [{ denom: `STAKE`, amount: `1` }]
+          }
+        ]
+      }
+    }
+  ],
+  fee: { amount: [{ denom: ``, amount: `0` }], gas: `21906` },
+  signatures: null,
+  memo: ``
+}
+
+const signMessage = createSignMessage(stdTx, { sequence, accountNumber, chainId });
+const signature = signWithPrivateKey(signMessage, Buffer.from(wallet.privateKey, 'hex'));
+```
+
+### Create a ready to broadcast signed transaction from a message
```

Comment:
```suggestion
### Create and sign a message that is ready for broadcasting
```

Reply:
does a message become a transaction when it's signed? 🤔

Reply:
a transaction is a message wrapped in some meta data ;)

Reply:
```suggestion
### Create and sign a transaction from a message which then is ready to be broadcast
```